### ISSUE TITLE
Error to register VM: Error attempting to get plugin server address for RPC: Failed to dial the plugin server in 10s

### ISSUE THREAD

**Issue by cshaw5** [minishift/minishift#3069]
### General information

I followed the [Microsoft ](https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/setup-nat-network)link to setup an internal virtual switch and then followed [OKD](https://docs.okd.io/latest/minishift/using/static-ip.html)'s instructions to launch minishift with an IP address and network info.

I am not able to use an External Switch because it severely hampers my download speed. I've updated my network driver to the latest version.

  * Minishift version:  v1.28.0+48e89ed
  * OS: Windows 10 - 1803
  * Hypervisor: Hyper-V

### Steps to reproduce

  1.`  minishift config set hyperv-virtual-switch "internal-nat"`
  2. ```.\minishift.exe start  --network-ipaddress 192.168.10.10   --network-gateway 192.168.10.1  --network-nameserver 8.8.8.8  --show-libmachine-logs -v5```

### Expected
To log into openshift console

### Actual
It creates a virtual machine within Hyper-V, but I can't manage it via 'minishift cmds'. I have to either reboot or use Hyper-V Manager to turn it off.

### Logs

<script src="https://gist.github.com/cshaw5/6e5e27cb1e0778f43ca4fbb6216c86a0.js"></script>

**Comment by cshaw5** [comment]
Windows 10 hyper-v "Error attempting to get plugin server address for RPC" #1381 
Seemed to have a similar issue, but I've tried to rerun it. I get the same error ever time.

After the error occurs I can't stop or delete it using minishift.
```
PS C:\minishift> minishift ssh
Command failed: Error attempting to get plugin server address for RPC: Failed to dial the plugin server in 10s
```
```
PS C:\minishift> minishift stop
Error attempting to get plugin server address for RPC: Failed to dial the plugin server in 10s
```

**Comment by gbraad** [comment]
Are you somehow connected to a VPN... or do you have anti-virus software installed?

I am a little unsure about the startup. Did the deployment succeed? The gist provided is not correctly given.

**Comment by cshaw5** [comment]
Good morning - I did a minishift delete -f and minishift start, and even though it hadn't worked before... It worked this time. 

Thank you for taking the time to respond.

**Comment by soul2zimate** [comment]
FYI, I am seeing the same error with following Env:
minishift v1.27.0+5981f99
CDK v3.7.0-1
OS name: "linux", version: "4.20.7-200.fc29.x86_64", arch: "amd64", family: "unix"
qemu-kvm version is  3.0.0

I disabled firewall and there is no active VPN.

If I switched to virtualbox with `minishift start --vm-driver virtualbox`, it started. After this successful startup, it can start in both vm driver for the second time.

My VirtualBox Version 5.2.24_RPMFusion r128122
